Packs/JoeSecurity/Integrations/JoeSecurity/JoeSecurity.yml

```diff
@@ -29,12 +29,12 @@ configuration:
   name: proxy
   required: false
   type: 8
-description: Sandbox Cloud
-display: Joe Security
+description: Deprecated. No available replacement.
```

Comment:
Isn't Joe Security v2 the available replacement for that integration?

Reply:
You are correct, fixed it 👍